repos.d/nix/nur.yaml

```diff
@@ -0,0 +1,29 @@
+###########################################################################
+# NUR
+###########################################################################
+- name: nur
+  sortname: nix-unstable~nur  # place right after Nix unstable
+  type: repository
+  desc: NUR combined
+  statsgroup: nix
+  family: nix
+  ruleset: [nix]
+  color: '7eb2dd'
+  minpackages: 100
+  default_maintainer: fallback-mnt-nur@repology
+  sources:
+    - name: packages-nur.json
+      fetcher:
+        class: FileFetcher
+        url: https://raw.githubusercontent.com/nix-community/nur-search/master/data/packages.json  # nur-search provides the aggregate json
+      parser:
+        class: NixJsonParser
+  repolinks:
+    - desc: NUR home
+      url: https://nur.nix-community.org/
+    - desc: GitHub repository
+      url: https://github.com/nix-community/NUR
+  packagelinks:
+    - type: PACKAGE_RECIPE
+      url: 'https://github.com/nix-community/nur-combined/blob/master/{posfile}#L{posline}'
```

Comment:
This produces broken URLs. The dump contains ready to use URLs but they are broken (trimmed up to repository path) sometimes as well.

Reply:
deactivated that part, like it's done for Chaotic-AUR

Reply:
Upon further investigation all position-URLs in the dump look fine to me.

Reply:
> Upon further investigation all position-URLs in the dump look fine to me.

`curl -s https://raw.githubusercontent.com/nix-community/nur-search/master/data/packages.json | grep \"position\" | grep -v \#`

Reply:
Ah, I misunderstood your first comment. These are really tricky ones, e.g. for `nur.repos.arc.packages.imv-develop` the JSON points to `https://github.com/nix-community/nur-combined/tree/master/repos/arc` but should actually point to [https://github.com/nix-community/nur-combined/blob/master/repos/arc/pkgs/customized/default.nix#L92](https://github.com/nix-community/nur-combined/blob/b48688290f4168bc1980c99caa2220f383ea90a5/repos/arc/pkgs/customized/default.nix#L92), which is a function that transforms the upstream package from nixpkgs and includes the result in NUR.
I think these packages should be filtered out.